### PR TITLE
Enemy behaviour

### PR DESCRIPTION
**Pull request by MadStark**
image

### CODE REVIEW

File: Assets/Scripts/Map/MapManager.cs [hunk 1]
```diff
@@ -231,13 +230,28 @@ private Vector3 GetRoomCenter(Room r)
...
+		        try
+		        {
+			        postProcessor.Execute(this);
+		        }
+		        catch (Exception e)
+		        {
+			        Debug.LogException(e);
+		        }
+	        }
+        }
```
Comment (Xwilarg) [inline Assets/Scripts/Map/MapManager.cs]: Indentation issue

File: Assets/Scripts/Player/PlayerBehaviour.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
```
Comment (Xwilarg) [inline Assets/Scripts/Player/PlayerBehaviour.cs:1]: Why not using PlayerManager.cs ?

File: Assets/Scripts/Player/PlayerController.cs [hunk 3]
```diff
@@ -31,6 +32,9 @@ public class PlayerController : MonoBehaviour
 		[SerializeField]
 		private int _gunForce;
 
+		public Camera Camera => _fpsCamera;
+		public bool IsGrounded { get; private set; }
```
Comment (Xwilarg) [inline Assets/Scripts/Player/PlayerController.cs]: What is this for?

File: Assets/Scripts/Player/PlayerController.cs.orig [hunk 4]
```diff
@@ -0,0 +1,193 @@
+<<<<<<< Updated upstream
```
Comment (Xwilarg) [inline Assets/Scripts/Player/PlayerController.cs.orig]: Please don't push that

File: Assets/Scripts/Player/PlayerManager.cs.orig [hunk 5]
```diff
@@ -0,0 +1,50 @@
+using System.Collections.Generic;
```
Comment (Xwilarg) [inline Assets/Scripts/Player/PlayerManager.cs.orig]: Please don't push it

File: Assets/Scripts/Player/PlayerManager.cs [hunk 6]
```diff
@@ -1,20 +1,45 @@
+using System.Collections.Generic;
 using Bug.Menu;
 using TMPro;
 using UnityEngine;
 
 namespace Bug.Player
 {
+	[DefaultExecutionOrder(-100)]
```
Comment (Xwilarg) [inline Assets/Scripts/Player/PlayerManager.cs]: I don't think we should keep that

**Review by Xwilarg — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Player/PlayerController.cs (hunk 3 above)
Comment (MadStark) [reply Assets/Scripts/Player/PlayerController.cs]: Exposing useful properties!

**Review by Xwilarg — APPROVED** [review]

**Pull request merged**